src/components/MetadataForm/MetadataForm.test.tsx

```diff
@@ -24,7 +23,7 @@ const props: Props = {
 const MetdadataColumnHeadersMock = mocked(MetdadataColumnHeaders);
 
 describe('<MetadataForm />', () => {
-	const render = (customProps: Partial<ComponentProps<typeof JsonForms>> = {}) => {
+	const render = (customProps: Partial<ComponentProps<typeof MetadataForm>> = {}) => {
```

Comment:
Considering the Props type is available within this repository it seems to be best practice to import the Props type rather than extracting it with ComponentProps.

Reply:
Added